package.json

```diff
@@ -1,92 +1,106 @@
 {
-    "name": "apify-client",
-    "version": "1.3.0",
-    "description": "Apify API client for JavaScript",
-    "main": "src/index.js",
-    "keywords": [
-        "apify",
-        "api",
-        "apifier",
-        "crawler",
-        "scraper"
-    ],
-    "author": {
-        "name": "Apify",
-        "email": "support@apify.com",
-        "url": "https://apify.com"
-    },
-    "contributors": [
-        "Jan Curn <jan@apify.com>",
-        "Marek Trunkát <marek@apify.com>",
-        "Ondra Urban <ondra@apify.com>",
-        "Jakub Drobník <jakub.drobnik@apify.com>"
-    ],
-    "license": "Apache-2.0",
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/apifytech/apify-client-js"
-    },
-    "bugs": {
-        "url": "https://github.com/apifytech/apify-client-js/issues"
-    },
-    "homepage": "https://apify.com/docs/sdk/apify-client-js/latest",
-    "files": [
-        "src",
-        "dist"
-    ],
-    "browser": {
-        "agentkeepalive": false,
-        "fs": false,
-        "os": false,
-        "stream": false,
-        "util": false,
-        "zlib": false
-    },
-    "scripts": {
-        "build": "npm run clean && npm run build-browser",
-        "build-docs": "npm run build-toc && node docs/build_docs.js",
-        "build-toc": "markdown-toc docs/README.hbs -i",
-        "test": "npm run build && jest",
-        "prepare": "npm run build",
-        "prepublishOnly": "(test $CI || (echo \"Publishing is reserved to CI!\"; exit 1))",
-        "clean": "rm -rf dist",
-        "lint": "eslint src test",
-        "lint:fix": "eslint --fix src test",
-        "build-browser": "webpack"
-    },
-    "dependencies": {
-        "@apify/consts": "^1.0.0",
-        "@apify/log": "^1.0.0",
-        "agentkeepalive": "^4.1.3",
-        "async-retry": "^1.3.1",
-        "axios": "^0.21.1",
-        "content-type": "^1.0.4",
-        "ow": "^0.23.0"
-    },
-    "devDependencies": {
-        "@apify/eslint-config": "^0.1.2",
-        "@babel/cli": "^7.11.6",
-        "@babel/core": "^7.11.6",
-        "@babel/preset-env": "^7.11.5",
-        "@babel/register": "^7.11.5",
-        "@types/express": "^4.17.8",
-        "@types/fs-extra": "^9.0.4",
-        "@types/jest": "^26.0.14",
-        "@types/node": "^14.11.5",
-        "apify": "^1.0.0",
-        "babel-loader": "^8.1.0",
-        "body-parser": "^1.19.0",
-        "brotli-webpack-plugin": "^1.1.0",
-        "compression": "^1.7.4",
-        "eslint": "^7.10.0",
-        "express": "^4.17.1",
-        "fs-extra": "^9.0.1",
-        "jest": "^26.5.0",
-        "jsdoc-to-markdown": "^6.0.1",
-        "markdown-toc": "^1.2.0",
-        "morgan": "^1.10.0",
-        "puppeteer": "^5.5.0",
-        "webpack": "^4.44.2",
-        "webpack-cli": "^3.3.12"
-    }
+	"name": "apify-client",
+	"version": "1.3.0",
+	"description": "Apify API client for JavaScript",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
```

Comment:
i'd be careful with the ESM support. i had to remove it from the apify-shared packages as it caused issues when used in downstream with webpack (it was easier to just remove it, not gonna lie, i am a webpack noobie, so i might just have missed something)

it might as well be caused by webpack 4 but as we are talking about our internal packages, we cant just force others in the company to upgrade webpack to v5 just to be able to use new packages that we ourselves create.

i'll try to look up what was the exact issue

Reply:
Could you check if https://github.com/apify/apify-client-js/pull/185#discussion_r688477222 is valid? It's what we do in other personal projects and it should just work..
Also hrm..yeah that would be a problem, upgrading to webpack 5 was easy here, I don't know how much it'll affect other company projects. Needs testing 😓

Reply:
ok now i remember - the thing was that guys are using cypress and it was not possible to override the webpack config it uses.

it was not even possible to read the consts, here is the failed build and one of the errors:

> Cannot read property 'ACTOR_BASE_DOCKER_IMAGES' of undefined

https://github.com/apify/apify-web/pull/1169/checks?check_run_id=2847671842

the workaround i came accross was `mainFields: ['browser', 'main', 'module'],` in the webpack config, which basically prioritizes the CJS version instead of ESM one - but apparently it was not possible to do so via cypress. 

(we can definitely keep it here for now, and remove it as a last resort if we encounter similar issues in downstream)

Reply:
> Could you check if #185 (comment) is valid? It's what we do in other personal projects and it should just work..

yes, @mnmkng is talking about the very same thing. but we did not fix it, the "fix" was to remove the ESM support from apify-shared.

https://github.com/apify/apify-shared-js/pull/199

i wouldn't be surprised if its a webpack 4 issue, also havent tested that myself extensively

Reply:
I mean worst case, I can remove the esm parts for now, as I can't test out the new version locally to see if it works or not. Thoughts?

Reply:
Please update changelog and version and we can merge it. It will release beta and we can test it.

Reply:
Done 👍